checkers/rules/rules.go

```diff
@@ -4,6 +4,9 @@ import (
 	"github.com/quasilyte/go-ruleguard/dsl"
 )
 
+// Bundle - used to import rules by external packages
+var Bundle = dsl.Bundle{}
```

Comment:
Unfortunately, we're using rules embedding in go-critic for builtin rules.
This means we can't export rules normally:
https://github.com/quasilyte/go-ruleguard/issues/355

This is not a hard issue to fix in ruleguard, but right now it's not an option to use bundle vars in linters that use a precompile step.